CHANGELOG.md

```diff
@@ -39,8 +39,25 @@ All notable changes to this project will be documented in this file.
 * ### `nidcpower` (NI-DCPower)
     * #### Added
         * Pass Python interpreter information if the driver runtime version supports it. This is used by NI in order to better understand client usage.
-        * Enums added:
-            * `CurrentLimitBehavior`
+        * API parity with NI-DCPower 2023 Q1.
+            * Properties added:
+                * `measure_complete_event_output_behavior`
+                * `measure_complete_event_toggle_initial_state`
+                * `sequence_engine_done_event_output_behavior`
+                * `sequence_engine_done_event_toggle_initial_state`
+                * `sequence_iteration_complete_event_output_behavior`
+                * `sequence_iteration_complete_event_toggle_initial_state`
+                * `source_complete_event_output_behavior`
+                * `source_complete_event_toggle_initial_state`
```

Comment:
All these are modeled after NI-FGEN's API.
In `nifgen` we made a conscious decision **to omit** the corresponding attributes. I don't know why / if the `nidcpower` rationale should be different. I think this warrants a bit of discussion.